src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,21 @@
+<%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+    <t:base>
+
+      <h1> Hello! ${username} </h1>
```

Comment:
Sorry to bug you about this, but can we update this with the new template code? I will merge the PR for it into master now, and you can merge master into your branch to get the updates

Reply:
- [ ] Update to new templating style, and use scriptlet to retrieve username: `<%= request.getParameter("username") %>`

Reply:
okay, thanks!